Networking/BLIP/BLIPConnection.cc

```diff
@@ -187,8 +187,7 @@ namespace litecore { namespace blip {
         // websocket::Delegate interface:
         virtual void onWebSocketConnect() override {
             _timeOpen.reset();
-            _connection->connected();
-            onWebSocketWriteable();
+            enqueue(FUNCTION_TO_QUEUE(BLIPIO::_onWebSocketConnect));
```

Comment:
This is not related to the title of the CBL ticket. Perhaps add a note in the commit comment.

Reply:
Thanks, will do